.github/workflows/codegen.yml

```diff
@@ -23,15 +23,13 @@ jobs:
     - uses: actions/checkout@v2
     - name: update_rust
```

Comment:
Nit: I'd highly recommend we use the `actions-rs/toolchain@v1` instead here (and in other actions yaml files)